mantidproject/mantid
# Disable editing on Events and Settings Reflectometry GUI tabs during processing.

Makes changes required to disable the corresponding group in the events and in the settings tab when processing is taking place for said group in the runs tab.

**To test:**
1. Open the Reflectometry GUI `Interfaces->ISIS Reflectometry`
2. Load a big table (e.g. `\\olympic\babylon5\Public\GemmaGuest\issues\21674_disable_editing_on_refl_gui\INTER_NR_test3.tbl`) and start processing the rows.
3. Switch to the `Settings` and `Event Handling` Tabs and ensure that the controls are disabled for the group in which you are processing.
4. Ensure that the settings for the other group remain enabled.
5. Pause the processing and ensure that the controls are re-enabled.
6. Resume processing and ensure that the controls are disabled.
7. When processing is finished ensure that the controls which were previously disabled are now enabled again.
8. Check that the SANS v2 GUI can be opened without errors.



Fixes #21674 . 

**Release Notes** 
*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
